js/hitbtc3.js

```diff
@@ -14,7 +14,7 @@ module.exports = class hitbtc3 extends Exchange {
             'pro': true,
             'has': {
                 'cancelOrder': true,
-                'CORS': false,
+                'CORS': undefined,
```

Comment:
In this case, we should test the CORS-capability of the exchange API and set this to its real value.

Reply:
yes, as turns out HitBTC doesnt have CORS permitted.